test/java/tests/PageCreationTest.java

```diff
@@ -0,0 +1,21 @@
+package tests;
+
+import org.testng.Assert;
+import pages.LoginPage;
+import org.testng.annotations.Test;
+import pages.NewPageEditor;
+
+public class PageCreationTest extends LoginPage {
+
+
+    @Test
+    public void testPageCreation()  {
+        NewPageEditor wordPressAdminpage = new LoginPage()
+                .openLoginPage()
+                .signIn(email, password)
+                .chooseActionToCreateANewPage()
+                .openPageEditor()
+                .createPageInTheEditor();
+        Assert.assertNotEquals(wordPressAdminpage, null);
```

Comment:
it's not a test which verifies whether an object representing a page is returned.
It should be something real. otherwise it's not user behavior, not requirements check, just abstract code thing.
If the form was populated, just check the result of its creation. It could be:
-the more common is to check whether population was successful - form which was filled is no more displayed, some success message is shown.
advanced checks:
-some result was receive to a user - email etc
-the result of populating the form could be observer from history, some place in user account etc

The goal of an assert is to check that after some actions we received some result expected for the user. Receive Result message of user action - user behavior, but check whether some abstract code object was created - no.